mailbag/formats/mbox.py

```diff
@@ -23,9 +27,10 @@ def account_data(self):
     def messages(self):
         for mail in self.data.itervalues():
             try:
+                parent = str(Path(self.file).parent)
                 message = Email(
                     Message_ID=mail['Message-ID'],
-                    Email_Folder=mail['Email-Folder'],
+                    Email_Folder=helper.emailFolder(self.dry_run,self.mailbag_name,self.format_name,parent,self.file),
```

Comment:
Still digging though this but helper.emailFolder isn't returning what I expect. for the account.mbox sample data we are using, it should return the string "Inbox", but its empty. Remember, for PST and MBOX, this folder structure is included in the file, so I don't think a cross-parser helper method will work. You this approach might be helpful for EML and MSG, since we're reading the directory structure for both, but for mbox, the parser is going to have to use [list_folders()](https://docs.python.org/3/library/mailbox.html#mailbox.Maildir.list_folders) or similar, much like the PST parser is doing with [number_of_sub_folders](https://github.com/UAlbanyArchives/mailbag/blob/develop/mailbag/formats/pst.py#L31) and [get_sub_folder](https://github.com/UAlbanyArchives/mailbag/blob/develop/mailbag/formats/pst.py#L32).